src/config/options.js

```diff
@@ -84,7 +84,7 @@ export const options = {
       }
     },
     // client-only queries
-    clientOnlyQueries: ['empty', 'filter_order', 'p', 'size', 'sortby'],
+    clientOnlyQueries: ['empty', 'filter_order', 'sortby'],
```

Comment:
I'll file an issue to remove the client-only queries `empty` and `sortBy` for a future sprint item 👍